test/src/test/java/hudson/model/FreeStyleProjectTest.java

```diff
@@ -113,7 +116,7 @@ public void customWorkspaceAllocation() throws Exception {
     @Issue("JENKINS-3997")
     public void customWorkspaceVariableExpansion() throws Exception {
         FreeStyleProject f = j.createFreeStyleProject();
-        File d = new File(j.createTmpDir(), "${JOB_NAME}");
+        File d = new File(tempFolder.newFolder().getPath(), "${JOB_NAME}");
```

Comment:
No need to convert the first argument to a string:

```suggestion
        File d = new File(tempFolder.newFolder(), "${JOB_NAME}");
```